src/browser/index.js

```diff
@@ -42,17 +43,40 @@ if (isReload) {
 document.body.classList.toggle('use-native-titlebar',
                                Runtime.useNativeTitlebar());
 
+const replay =
+  () => {
+    const request = new XMLHttpRequest();
+    // We use sync XHR as we have nothing else to render otherwise.
+    request.open
+    ( 'GET'
+    , String(Runtime.env.replay)
+    , false
+    );
+    request.overrideMimeType('text/plain');
+    request.send();
+
+    if (request.status === 200 || request.status === 0) {
+      return [JSON.parse(request.responseText), Effects.none]
```

Comment:
Instead, can you use   `xhr.responseType = 'json'` and then `return xhr.response`?